### ISSUE TITLE
[Screen reader - Make Code - Homepage] Narrator focus moves to hidden element in scan mode under Homepage.

### ISSUE THREAD

**Issue by sonimishra** [microsoft/pxt-microbit#2363]
**User Experience:**
Screen reader user will be misguided if narrator focus moves to hidden element as user will be confused and also screen reader user might expect some functionality over hidden element if narrator provides some hidden control which is not appearing visually.

**Test Environment:**
Version 1903 (OS Build 18867.1000)
Platform: Edge
Screen reader: Narrator

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/beta#
2. Home page will open.
3. Set to scan mode by using Caps + Space keys. 
4. Navigate to the various controls appearing under "Science, Tools, Turtle" section in Home page.
6. Verify various control in scan mode by using narrator.

**Actual Result:**
Narrator focus moves to hidden control while user navigates in scan mode under "Science, Tools, Turtle" section on Home page and due to this, screen reader announces "See more button" and "See previous button" which is visually not appearing on homepage.

**Expected Result:**
Narrator focus should not move to hidden control while user navigates in scan mode under "Science, Tools, Turtle" section on Home page and screen reader should not announces "See more button" and "See previous button" if control is not visually appearing on homepage.

**MAS Reference:**
[MAS 4.2.10 - Tracking of Focus and Selection Attributes (33J)
](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7bb24201d1-2276-4471-8183-e1e5353031ad%7d)

[Accessibility Insights!]((https://microsoft.sharepoint.com/teams/msenable/Pages/PageNotFoundError.aspx?requestUrl=https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx
) - Identify accessibility bugs before check-in and make bug fixing faster and easier.)

[MAS4.2.10_Homepage_Science.zip](https://github.com/microsoft/pxt-microbit/files/3476554/MAS4.2.10_Homepage_Science.zip)

**Comment by sonimishra** [comment]
Issue has been verified and fixed.
Hence closing the bug.
